pkg/agent/controller/networkpolicy/cache.go

```diff
@@ -80,7 +80,7 @@ type rule struct {
 
 // hashRule calculates a string based on the rule's content.
 func hashRule(r *rule) string {
-	hash := sha1.New()
+	hash := sha1.New() // #nosec G401: not used for security purposed
```

Comment:
```suggestion
	hash := sha1.New() // #nosec G401: not used for security purposes
```

Reply:
what does this comment mean btw?

Reply:
ignore this.. i googled .. for anyone who wants to know -> https://github.com/securego/gosec

Reply:
gosec flags hash functions that can be "reversed" and that should not be used for crypo (digital signature, message integrity), but that's not our usage here